frontend/taipy-gui/src/components/Taipy/tableUtils.tsx

```diff
@@ -472,14 +475,27 @@ export const EditableCell = (props: EditableCellProps) => {
                 {edit ? (
                     colDesc.type?.startsWith("bool") ? (
                         <Box sx={cellBoxSx}>
+                            lightBool ? (
+                            <input
+                                type="checkbox"
+                                checked={val as boolean}
+                                title={val ? "True" : "False"}
```

Comment:
What about using the property as the place to define those texts?
Like... `|use_checkbox=False;True|` or
```
check_values = ("Off", "On")
|use_checkbox={check_values}|
```
If `|use_checkbox|` (`tgb.table(..., use_checkbox=True)`) then False/True would be the 'default`...

Reply:
those 2 are not related
the title is defined on the switch control too

Reply:
[#1826](https://github.com/Avaiga/taipy/issues/1826)